src/System.Resources.Writer/src/System.Resources.Writer.csproj

```diff
@@ -9,8 +9,9 @@
   <ItemGroup>
     <Compile Include="System\Resources\__FastResourceComparer.cs" />
     <Compile Include="System\Resources\IResourceWriter.cs" />
-    <Compile Include="System\Resources\ResourceWriter.cs" />
-    <Compile Include="System\Runtime\Versioning\MultitargetingHelpers.cs" />
+    <Compile Include="System\Resources\ResourceWriter.core.cs" />
```

Comment:
We prefer to use camel case for names. This would make it `.Core.cs`. (There are several files called `.Core.cs` in the repo, but no `.core.cs`)

When I see "Core", I think about it as the central part of the implementation. This file is not the central part of the implementation. The file should have a comment with an explanation at least.

Reply:
got it, will update.